packages/react-ui-ag/src/Gmap/Marker.js

```diff
@@ -1,7 +1,7 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import { Marker as Gmarker } from '@rentpath/react-ui-core'
-import { markerIcon } from './markerIcons'
+import { markerRedDotIcon } from './markerIcons'
```

Comment:
Do the marker icons live in react-ui-ag now?

Reply:
Can you help me understand the decision to change the icon names to "RedDot" and "BlackDot"? If we decide to change the marker color some day, won't this make it harder?

Reply:
@bethstevic there encoded in the urls now so we don't have to worry about them
@mpelzsherman I changed them because otherwise you can't look at them and easily tell what they are.  The logic here was that we may want to test other markers at some point and we can't do that unless we clearly specify the marker intent.  We could always add something to this like `markerBlueDotIcon`. Either way I'm fine with whatever we decide.